From 7d7083a6893ba4197f0d0ff92b139f0c226878c9 Mon Sep 17 00:00:00 2001
From: Fabio Berton <fabio.berton@ossystems.com.br>
Date: Thu, 12 Apr 2018 16:19:42 -0300
Subject: [PATCH] rules.d/01-stick.rules: Change rule to use with UpdateHub
Organization: O.S. Systems Software LTDA.

Change udev rule to not run when USB device has label updatehub or UPDATEHUB.

Signed-off-by: Fabio Berton <fabio.berton@ossystems.com.br>
---
 stick/etc/udev/rules.d/01-stick.rules | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/stick/etc/udev/rules.d/01-stick.rules b/stick/etc/udev/rules.d/01-stick.rules
index ae620a4..5cf6862 100644
--- a/stick/etc/udev/rules.d/01-stick.rules
+++ b/stick/etc/udev/rules.d/01-stick.rules
@@ -1,3 +1,2 @@
-SUBSYSTEM=="block", ACTION=="remove", KERNEL=="sd*", NAME="%k", RUN+="/usr/lib/stick/stick-umount.sh"
-SUBSYSTEM=="block", ACTION=="add", KERNEL=="sd?1", NAME="%k", RUN+="/usr/lib/stick/stick-mount.sh"
-
+ENV{ID_FS_LABEL}!="updatehub|UPDATEHUB", ENV{DEVTYPE}=="partition", ACTION=="remove", RUN+="/usr/lib/stick/stick-umount.sh"
+ENV{ID_FS_LABEL}!="updatehub|UPDATEHUB", ENV{DEVTYPE}=="partition", ACTION=="add", RUN+="/usr/lib/stick/stick-mount.sh"
-- 
2.14.2

